scripts/status.js

```diff
@@ -100,18 +100,20 @@ var date = new Date();
 var zerotime = date.getTime() - ((numBGs * 5) * 600);
 var zero_x = numBGs + 5;
 for (var i = 0; i <= numBGs; i++) {
```

Comment:
The underlying bug is this `<=` should be a `<`.  Want to fix that as well (and leave your null check too)?